src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -173,6 +174,8 @@ public double getVelocityInput() {
   public void windFlywheel(double rpm) {
 
     // Winds Flywheel using PID control to passed rpm
+    Shuffleboard.addEventMarker("FlyWheel Starts", EventImportance.kHigh);
```

Comment:
Shuffleboard.addEventMarker("FlyWheel Starts" + rpm, EventImportance.kHigh);

That way, the point in time RPM is logged at the same time as the event.  

There is already a dashboard value for RPM

Reply:
Done. I also changed ballCount that was on SmartDashboard to being on Shuffleboard.